website/azurerm.erb

```diff
@@ -688,6 +688,10 @@
             <li<%= sidebar_current("docs-azurerm-resource-autoscale-setting") %>>
```

Comment:
now there's multiple items in here, can we update this to: `docs-azurerm-resource-monitor`